cfy_manager/components/postgresql_server/config/etcd.conf

```diff
@@ -0,0 +1,15 @@
+ETCD_LISTEN_PEER_URLS='https://{{ manager.private_ip }}:2380'
+ETCD_LISTEN_CLIENT_URLS='https://localhost:2379,https://{{ manager.private_ip }}:2379'
+ETCD_INITIAL_ADVERTISE_PEER_URLS='https://{{ manager.private_ip }}:2380'
+ETCD_INITIAL_CLUSTER='{% for node in postgresql_server.cluster.nodes -%}etcd{{ node.replace('.', '_') }}=https://{{ node }}:2380,{% endfor %}'
```

Comment:
trailing comma is allowed?

Reply:
Yup, possibly even required. All the examples I saw had it, so I left it in.